src/components/icon/icon.spec.js

```diff
@@ -148,11 +148,36 @@ describe("Icon component", () => {
 
       it("renders properly colored Icon when hovered", () => {
         const wrapper = mount(<Icon color={color} type="message" />);
+        expect(wrapper.find(StyledIcon)).not.toHaveStyleRule(
+          "color",
+          shade(0.2, toColor(baseTheme, color)),
+          { modifier: ":hover" }
+        );
+      });
+
+      it("takes precedence over iconColor and renders properly colored Icon with tooltip", () => {
+        const wrapper = shallow(
+          <Icon type="home" color={color} tooltipMessage="tooltip message" />
+        );
+
+        assertStyleMatch(
+          {
+            color: toColor(baseTheme, color),
+          },
+          wrapper.find(StyledIcon).dive().dive().dive()
+        );
+      });
+
+      it("renders properly colored Icon with tooltip when hovered", () => {
+        const wrapper = shallow(
+          <Icon type="home" color={color} tooltipMessage="tooltip message" />
+        );
+
         assertStyleMatch(
           {
             color: shade(0.2, toColor(baseTheme, color)),
           },
-          wrapper.find(StyledIcon),
+          wrapper.find(StyledIcon).dive().dive().dive(),
```

Comment:
We didn't previously need to use `dive`, we try to avoid because it makes the test more brittle.

Reply:
changed, thanks.